pkg/runners/zero/zero_benchmark_test.go

```diff
@@ -3,6 +3,9 @@ package zero
 import (
 	"math"
 	"testing"
+
+    hintrunner "github.com/NethermindEth/cairo-vm-go/pkg/hintrunner"
```

Comment:
Does your editor/IDE auto-apply the gofmt on save?
I noticed a couple of lines that look like they're indented with spaces.

Reply:
I'll check that